lib/solr_wrapper/instance.rb

```diff
@@ -103,35 +103,31 @@ def restart
     ##
     # Check the status of a managed Solr service
     def status
```

Comment:
As it is only ever called locally and all those calls use started? instead, it doesn't matter (at least the specs don't care, so that's goog enough for me, esp. if this is all private anyway).